common/persistence/nosql/nosqlplugin/cassandra/gocql/session.go

```diff
@@ -87,12 +96,16 @@ func (s *session) refresh() {
 
 	if time.Now().UTC().Sub(s.sessionInitTime) < sessionRefreshMinInternal {
 		s.logger.Warn("gocql wrapper: too soon to refresh gocql session")
```

Comment:
I know it is not a part of this PR but could you also make this error message better, explicitly saying that last refresh was less than 5 seconds ago and skipped this time.